docs/solutions/rhel74_image/README.md

```diff
@@ -1,7 +1,45 @@
 This example provides a method to generate a RHEL 7.4 image for use by both VM and BM shapes.
+
+UPDATED 23 May 2018!  Please read below!
```

Comment:
I will propose doing this as a CHANGELOG.md, similar to what we have at https://github.com/oracle/terraform-provider-oci/blob/master/CHANGELOG.md. Then remove this from the readme and maybe add a single line with something like "See the changelog for recent updates."